1904-11-24.xml

```diff
@@ -0,0 +1,1945 @@
+                        few streets in Cario, but it appears that the efforts have been relaxed for
+                        there are many streets in the Capital which are innocent of any designation
+                        whatsoever. We may instance three or four streets running off the
+                        Kasr-el-Ainy-road, on the left as one goes towards the hospital from
+                        Kasr-el-Nil Bridge. These streets are nameless, and therefore the regulation
+                        of the Post office cannot be complied with. It has been maliciously rumoured
+                        that the reason why so many streets are unnamed is because of the jealousy
+                        among officials, who cannot come to any agreement among themselves and to
+                        which of them shall be immortalized by christening the streets with their
+                        own names. This is ridiculous, for the modesty of our official classes is
+                        proverbial. But we fail to see any reason why the names of prominent
+                        Anglo-Egyptians should not figure on the street as well as those of native
+                        notables, such as Idris Bey Ragheb, whose name adorns a street near El
+                        Daher, and also in other cases. At Alexandria we have streets named after
+                        Goussio, Adib, Antonisdis, Gordon, Mariette, and numerous other local
+                        notabilties. Surely there are plenty of officials belonging to the Public
+                        Works Department or the Ministries of Justice and the Interior whose names
+                        would be useful for these streets ! Such names as Garstin, Dunlop, or
+                        McIlwraith might fittingly be adapted to street nomenclature. As the roads
+                        we have mentioned are in the neighbourhood of the new schools by the
+                        Kasr-el-Ainy-road it would be appropriate to use some of the names of the
+                        high officials in the Ministry of Public Instruction, such as that of Dr.
+                        Dunlop, Yacoub Pasha Artin, etc. The opportunities  thus afforded for
+                        immortalizing many names whose owners are closely connected with the
+                        development of modern Egypt would be unllimited, as new streets are
+                        springing up daily. Another plan, in case of disagreement, would be to put
+                        the matter up to an auction, as there must be many who would be only too
+                        eager to see their names inscribed on the walls of Cairo for all time. The
+                        sum realised should be of great utility in swelling the revenues of the
+                        Tanzim Department, which pleads poverty as an excuse for neglecting its
+                        duties towards the public.</p>
```

Comment:
Put a `</div>` at the end of this line, to close this item. Otherwise, the following item is *inside* this item.